cypress/integration/Dictionary/createDictionary.feature

```diff
@@ -0,0 +1,110 @@
+Feature: Dictionary Management
+
+  Scenario: A user to create a new dictionary
+    Given the user is logged in
```

Comment:
Since all of these tests assume the user is logged in, this should be something that is setup in the background [as is demonstrated in the "Login Page"](https://github.com/openmrs/openmrs-ocl-client/blob/4d5150a2a475cba0e7876bf77c1aa39c54aaee75/cypress/integration/login.feature#L2) feature. I.e., you should have something like:

```gherkin
Background:
  Given the user is not logged in
```